grid2op/Opponent/OpponentSpace.py

```diff
@@ -31,7 +31,8 @@ class OpponentSpace(object):
     budget_per_timestep: ``float``
         The increase of the opponent budget per time step (if any)
     """
-    def __init__(self, compute_budget, init_budget, opponent, budget_per_timestep=0., action_space=None):
+    def __init__(self, compute_budget, init_budget, opponent, budget_per_timestep=0., action_space=None,
```

Comment:
Change that where the OpponentSpace is initialized too :-) [the parameters attack_duration=12*4, attack_cooldown=12*24 ] should be exposed in the environment and not deep down here only.

Reply:
Done